js/render.js

```diff
@@ -0,0 +1,22 @@
+import {createPhotoCollection} from './mock.js';
+
+const picturesContainer = document.querySelector('.pictures');
+
+const pictureTemplate = document.querySelector('#picture')
+	.content
+	.querySelector('.picture');
+
+const photos = createPhotoCollection();
+
+const photosFragment = document.createDocumentFragment();
+
+photos.forEach(({url, description, likes, comments}) => {
+	const photoElement = pictureTemplate.cloneNode(true);
+	photoElement.querySelector('.picture__img').src = url;
+	photoElement.querySelector('.picture__img').alt = description;
```

Comment:
Всё-таки советую не вызывать функции дважды
```suggestion
	const imageElement = photoElement.querySelector('.picture__img');
	imageElement.src = url;
	imageElement.alt = description;
```